comfy_cli/command/run/preflight.py

```
@@ -0,0 +1,157 @@
+    if warnings and renderer.is_pretty():
+        for w in warnings:
+            pprint(f"[yellow]⚠ {w.get('field', '?')}: {w.get('message', '')}[/yellow]")
+
+
+def _fetch_object_info(host: str, port: int, timeout: int = 10) -> dict:
+    """Fetch object_info for partner-node detection + validation. Fail open."""
+    try:
+        from comfy_cli.cql.engine import _load_from_target
+
+        return _load_from_target(mode="local", host=host, port=port)
+    except Exception:  # noqa: BLE001 — fail open
```

Pylint (4.0.5)

[warning] Catching too general exception Exception

(W0718)